misk/web/tabs/loader/src/containers/LoaderContainer.tsx

```diff
@@ -1,58 +1,59 @@
 import { IMiskAdminTab, IMiskAdminTabs } from "@misk/common"
 import { NavSidebarComponent, NavTopbarComponent, NoMatchComponent } from "@misk/components"
+import { externals as MiskTabCommon } from "@misk/tabs" 
+import { RouterState } from "connected-react-router"
 import * as React from "react"
 import { connect } from "react-redux"
 import { Route, Switch } from "react-router"
 import { Link } from "react-router-dom"
 import { dispatchLoader } from "../actions"
 import { MountingDivComponent, ScriptComponent } from "../components"
-import { IState } from "../reducers"
+import { ILoaderState, IState } from "../reducers"
+import { IMultibinder } from "../utils/binder"
+
+interface IMiskTabs {
+  [app:string]: any
```

Comment:
I want you to explain me this syntax later . . . it’s the first one I’ve seen that isn’t self-explanatory

Reply:
For sure! Typescript interfaces took a bit of getting used to/